.github/workflows/minimal.yml

```
@@ -4,9 +4,9 @@ name: Minimal installation
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, support/2.x ]
```

```suggestion
    branches: [ support/v2]
```
I think we don't want this triggering on `main` (which will have its own GH actions files?)